### ISSUE TITLE
[BUG] Missing the browser name when generated snapshot file

### ISSUE THREAD

**Issue by done3t** [microsoft/playwright#8202]
**Context:**
- Playwright Version: 1.13.1 & 1.14.0
- Operating System: Mac
- Node.js version: 14.16
- Browser: Chromium

**Code Snippet**

```javascript
import { test, expect } from "@playwright/test";

test.only('basic test', async ({ page }) => {
    await page.goto('https://playwright.dev/')
    const input = await page.waitForSelector('.navbar__search-input')
    await input.type('Playwright')
    expect(await input.inputValue()).toBe('Playwright');
    expect(await page.screenshot()).toMatchSnapshot('a.png')
});
```

**Describe the bug**

![image](https://user-images.githubusercontent.com/49627244/129470623-b0aa4a2b-f9af-487c-aa0f-e714fccc8fd6.png)

I found that it lost the browser name when generated snapshot file name after I updated to 1.13.1 or 1.14.0
Is this a new feature?

**Extra Note**
Identical issue presents on Debian with Playwright 1.13.1 / 1.14.0

**Comment by dgozman** [comment]
@done3t Do you use `playwright.config` or pass `--browser=` command line option? If not, there was a bug fix in v1.13.2 that stopped adding the browser name when you don't specify anything in the config nor use the `--browser` option. Sorry for the inconvenience!

I'd suggest to create a [configuration file](https://playwright.dev/docs/intro#creating-a-configuration-file) to have a nice place for you test configuration.

Does this answer your question?

**Comment by done3t** [comment]
yes.
I had create a playwright.config, but didn't config any project or browser name, so that I thought it's a new issue.
thank you for your answer.
